src/backend/clearboard/main.py

```diff
@@ -47,3 +96,124 @@ async def post_picture(file: UploadFile = File(...)):
         with open(f"{path}/{file.filename}", 'wb') as f:
             shutil.copyfileobj(file.file, f)
         return {"message": file.filename}
+
+
+def image_to_base64(img: np.ndarray) -> bytes:
+    """ Given a numpy 2D array, returns a JPEG image in base64 format """
+    img_buffer = cv2.imencode('.jpg', img)[1]
+    return base64.b64encode(img_buffer).decode('utf-8')
+
+
+def get_image(image):
+    return image_to_base64(image)
+
+
+@app.get("/process")
+async def photo(roomName: str, process: str):
+    try:
+        img_cropped_path = './' + roomName + '/' + roomName + 'cropped.jpg'
+        original_img_path = './' + roomName + '/' + roomName + '.jpg'
+        processed_img_path = './' + roomName + '/' + roomName + process + ".jpg"
+        if os.path.exists(os.path.abspath(img_cropped_path)):
+            img_to_process = img_cropped_path
+        elif os.path.exists(os.path.abspath(original_img_path)):
+            img_to_process = original_img_path
+
+        if process == 'Color':
```

Comment:
you should normalize the machine name of your filters with snake_case : color, bw, contrast, original, superres
as make them constants. Also, "original" is the same as `else` so better don't materialize it and use the `None` value for this case?
You could also define a mapping of your filters to their methods so the code is simpler and it's easy to add a new filter? Something like:
```python
COLOR = "color"
BW = "bw"
CONTRAST = "contrast"
HIGHRES = "highres"
FILTERS = {
    COLOR: whiteboard_enhance,
    BW: black_n_white,
    CONTRAST: enhance_contrast,
    HIGHRES: high_resolution
}
```